src/utils.js

```diff
@@ -39,15 +39,25 @@ export function readableNumber(num) {
   } else if (num >= 10000) {
     return `${(num / 1000).toFixed(1)}k`;
   } else if (num < 10000) {
-    return digitGrouping(num);// using only digitGrouping because we only need to display any number below 10k with only a comma
+    return digitGrouping(num); // using only digitGrouping because we only need to display any number below 10k with only a comma
   }
   return num;
 }
 /**
- * 
+ *
```

Comment:
The description of the function should be in this line, not in `return`. See the function below.

Reply:
Done